src/install.rst

```diff
@@ -184,6 +184,29 @@ Set up a Nextstrain runtime
             .. include:: snippets/nextstrain-setup-conda.rst
 
 
+   .. group-tab:: Singularity (Apptainer)
+
+      Singularity is a container system freely-available for Linux platforms. It is commonly available on institutional HPC systems as an alternative to Docker, which is often not supported on such systems.
+
+      The Singularity project forked into two separate projects in late 2021: `SingularityCE <https://sylabs.io/singularity/>`__ and `Apptainer <https://apptainer.org>`__.  Either fork should work with Nextstrain CLI, as both projects still provide very similar interfaces and functionality via the ``singularity`` command.
+
+      .. note::
+
+         These instructions are for institutional HPC systems that have Singularity installed and configured. You may need to load it with ``module load singularity``. If you don't have Singularity available, consider using one of the other runtimes or continue by installing `SingularityCE <https://docs.sylabs.io/guides/3.0/user-guide/installation.html>`__/`Apptainer <https://apptainer.org/docs/admin/main/installation.html>`__.
+
+      1. Check that Singularity is available in your environment.
+
+         .. code-block:: bash
+
+            singularity --version
+
+      2. Set up the runtime:
+
+         .. code-block:: none
+
+            nextstrain setup --set-default singularity
```

Comment:
Tested on Fred Hutch's `rhino` and it didn't work. I created a separate issue: https://github.com/nextstrain/cli/issues/360